src/portable/nuvoton/nuc505/dcd_nuc505.c

```diff
@@ -465,6 +467,7 @@ void USBD_IRQHandler(void)
 
     if (bus_state & USBD_BUSINTSTS_DMADONEIF_Msk)
     {
+#if 0
```

Comment:
We should either remove these DMA code completely if DMA issue is not solvable, or defined an macro e.g `USE_DMA = 0`, with a bit of description  in which situation it doesn't work, like you put in the PR description. It will help us a lot after a while looking back at the DMA issue.